accelerator-core/src/main/java/com/tokbox/android/otsdkwrapper/wrapper/OTWrapper.java

```diff
@@ -754,6 +757,32 @@ public synchronized void setOutputSignalProtocol(SignalProtocol outputProtocol)
     }
   }
 
+  /**
+   * Return boolean indicating whatever or nor device has multiple cameras available. This allows you to easily
+   * show/hide "swap camera" button based on current device config.
+   * @return Information about multiple available cameras
+   */
+  public boolean hasMultipleCameras() {
```

Comment:
@igorwojda I am not sure if we should add this method in the OTWrapper API. It seems to be an action at custom application level.

Reply:
I agree it's not 100% opentok, but this facilitates usage and complements our SDK.

On one side we are giving customer ability to switch camera, on the other hand we don't provide any information if this action makes seanse for given device. We should be always thinking about complete usage scenarios (opentok and around) to simplify things and help devs as much as we can eg.

Current hevaviour:
1. Dev have to figure out if showing switch camera button makes seance (we are not helping here at all)
2. Actually swap camera

Proposal (this PR):
1. Check if there is more then one camera (this will easily allow dev to show or hide "swap camera" button)
2. Actually swap camera

Reply:
Thanks @igorwojda for the explanation! I think this explanation of the use case should be in the PR description to explain what this PR is doing.

IMHO I think it's a useful helper method, but I agree with @marinaserranomontes. I feel this is not required in the OT wrapper. In addition, If we add this method in this repo, we should consider to add it in the accelerator-core for iOS as well.